lib/spack/spack/test/spec_dag.py

```diff
@@ -853,12 +853,14 @@ def test_normalize_diamond_deptypes(self):
         self.check_diamond_deptypes(s)
         self.check_diamond_normalized_dag(s)
 
+    @pytest.mark.usefixtures('config')
```

Comment:
Changes in this file shouldn't be needed anymore as recently the `config` fixture has been added at class level.